### DISCUSSION TITLE
Dynamic footer to link to various social coding platforms

### DISCUSSION THREAD

**Discussion by gcharest** [gc-da11yn/gc-da11yn.github.io#64] [Inclusive Design (Open Discussions)]
Propose the use of template import in default layout to dynamically render the Contributing  footer with the right social coding platform.

For example, a site could have a designated value has it's default location (Github in the current example) but that variable could be overridden by someone reusing the footer and having their repo hosted on GitLab or Bitbucket.

**Comment by shawnthompson** [comment]
I think this should be moved to the [GCWeb](https://github.com/wet-boew/gcweb) project. Thoughts @pcwsmith?

**Comment by shawnthompson** [comment]
I'd like to add it to the WET (not GCWeb) base template.

Thoughts @duboisp @GormFrank?

![image](https://user-images.githubusercontent.com/2575266/130069886-5033c35c-0efb-4055-92bf-7d3dabf4747a.png)

So have one for Gitlab.com, GCCode (which uses Gitlab), GitBucket, GitOnIt (I made this one up)...

**Reply by duboisp** [reply]
Yes, but there is not really an official way to submit that pattern for a review by DTO yet. May be we will have that kind of process later when the GC design system project will be more mature.

We can probably have that pattern documented in GCWeb but it's usage will be upon site owner needs. For wet-boew, we can simply change the existing site footer.

**Reply by shawnthompson** [reply]
Let's change the footer in wet-boew template and go from there.

I'll open an issue in the repo and assign it to myself. Been too long since I worked on WET :)

**Reply by shawnthompson** [reply]
https://github.com/wet-boew/wet-boew/issues/9206

**Comment by gcharest** [comment]
If we can have this standardized and customizable based on the platform, it would make things easier indeed.
